### PR TITLE
fix: ore & stone types generation

### PR DESCRIPTION
**Pull request by KoshakMineDEV**

### CODE REVIEW

File: src/main/java/cn/nukkit/level/generator/object/ore/OreType.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public boolean spawn(ChunkManager level, NukkitRandom rand, int replaceId, int x
                                 double zVal = ((double) zSeg + 0.5D - scaleZ) / (randVec1 / 2.0D);
 
                                 if (xVal * xVal + yVal * yVal + zVal * zVal < 1.0D) {
-                                    if (level.getBlockIdAt(xSeg, ySeg, zSeg) != 0) {
+                                    if (level.getBlockIdAt(xSeg, ySeg, zSeg) == STONE) {
```
Comment (CoolLoong) [inline src/main/java/cn/nukkit/level/generator/object/ore/OreType.java:91]: why?Other types of stones should also be replaced,should not only replace stone

Comment (KoshakMineDEV) [reply src/main/java/cn/nukkit/level/generator/object/ore/OreType.java:91]: This types of stone is also generated by TypeOre. Currently, after @Buddelbubi world generator changes ores can spawn on the terrrain and in the water.

Comment (KoshakMineDEV) [reply src/main/java/cn/nukkit/level/generator/object/ore/OreType.java:91]: only one thing i need to check is ore generation in deeplstate

**Comment by KoshakMineDEV** [comment]
Now works 100% right

**Review by CoolLoong — APPROVED** [review]

File: src/main/java/cn/nukkit/level/generator/object/ore/OreType.java (hunk 1 above)
Comment (Buddelbubi) [reply src/main/java/cn/nukkit/level/generator/object/ore/OreType.java]: Oh sorry. Didn't thought about that

**Pull request merged**